backend/src/main/scala/com/softwaremill/bootzooka/http/model/headers/headers.scala

```diff
@@ -0,0 +1,86 @@
+package com.softwaremill.bootzooka.http.model.headers
```

Comment:
Right, maybe `api` isnt't the best name. It tries to group the "util" or "non-business" code which is later used for actually implementing the APIs. Do you have a better name suggestion? The rest of the packages are ok I think, grouped by functionality. And I also think that e.g. `RoutesSupport` and your `headers` should go into the same package - both are supporting code for actual routes implementation.

As this is intended to be a simple startup project, I would be cautious with adding any, let's say, "non-standard" specific patterns for organizing code, as then it often depends on the specific project how the code should be organized. So I would be aiming at something that doesn't surprise people too much :)